### PR TITLE
Add margin to the device automation form

### PR DESCRIPTION
**Pull request by ZephireNZ**
## Proposed change

Adds a small margin to the picker for device automations, so that the form with extra fields isn't jutted up against.

Before:
image

After:
image

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by piitaya** [comment]
This will add margin even is extra-fields are not displayed. 
We should use something like this : 
```css
:host :not(:last-child) {
  margin-bottom: 16px;
}
```

**Comment by bramkragten** [comment]
Or:

```css
ha-form {
    margin-top: 24px;
}

File: src/panels/config/automation/action/types/ha-automation-action-device_id.ts [hunk 1]
```diff
@@ -152,9 +152,14 @@ export class HaDeviceAction extends LitElement {
       display: block;
       margin-bottom: 16px;
```
Comment (ZephireNZ) [inline src/panels/config/automation/action/types/ha-automation-action-device_id.ts:153]: I just noticed this form is 16px margin but the other two are 24px.

Did we want to standardise the margin while we're at it?

Comment (piitaya) [reply src/panels/config/automation/action/types/ha-automation-action-device_id.ts:153]: Yes 👍

Comment (ZephireNZ) [reply src/panels/config/automation/action/types/ha-automation-action-device_id.ts:153]: So would that be changing them all to 24px or 16px?

Comment (bramkragten) [reply src/panels/config/automation/action/types/ha-automation-action-device_id.ts:153]: 24

**Review by bramkragten — APPROVED** [review]

**Pull request merged**